src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -51,7 +52,12 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
       {typeof picture === 'string' ? (
         <FastImage source={{uri: picture}} style={styles.picture} />
       ) : (
-        <FastImage source={picture} style={[styles.picture, styles.anonBgColor]} />
+        <AnonymousIcon
+          fill={COLORS.holyTosca}
+          width={dimen.normalizeDimen(20)}
+          height={dimen.normalizeDimen(20)}
+          marginRight={dimen.normalizeDimen(6)}
+        />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The way the anonymous icon is rendered has been changed. Instead of using a FastImage component with a source prop, an AnonymousIcon component is now used with a fill prop set to a color from the new `COLORS` constants. This change improves flexibility by allowing the color of the icon to be customized.
```diff
- <FastImage source={picture} style={[styles.picture, styles.anonBgColor]} />
+ <AnonymousIcon
+   fill={COLORS.holyTosca}
+   width={dimen.normalizeDimen(20)}
+   height={dimen.normalizeDimen(20)}
+   marginRight={dimen.normalizeDimen(6)}
+ />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->